lib/memoizable/method_builder.rb

```diff
@@ -38,6 +38,9 @@ def initialize(descendant, method)
 
     # The original method before memoization
     #
+    # @example
+    #    method_builder.original_method  # => :foo
+    #
```

Comment:
__TODO:__ Extract this into two commits:

- Add Memoizable::Memory YARD docs
- Add Memoizable::MethodBuilder#original_method YARD docs